input/core/src/commonMain/kotlin/majestic/editor/body/chunks/Image.kt

```diff
@@ -1,10 +1,18 @@
 package majestic.editor.body.chunks
 
+import androidx.compose.ui.graphics.painter.BitmapPainter
+
 
 data class Image(
     override val uid: Int,
-    val url: String?,
-    val caption: String?
+    val uri: String?,
+    val caption: String?,
+    val painter: BitmapPainter? = null,
```

Comment:
Adding a painter here makes this class not eligible to be sent to the server, we need to find a way to either update this class, or create another one